understanding/dragging-movements.html

```diff
@@ -56,213 +56,154 @@
                   </p> 
                   
                   
                </section>
                <section id="distinguishing-dragging-movements-from-path-based-pointer-gestures">
                   
-                  <h3>Distinguishing dragging movements from path-based pointer gestures</h3>
+                  <h3>ドラッグ動作と軌跡ベースのポインタジェスチャの区別</h3>
                   
                   
-                  <p>Dragging movements covered in this Success Criterion are pointer interactions where
-                     only the start- and endpoints matter. Once the pointer engages with a target, the
-                     direction of the dragging movement does not factor into the interaction until the
-                     pointer disengages the target. Since the dragging movement does not have an intermediate
-                     point, the dragging movement can go in any direction. Path-based gestures are covered
-                     in Success Criterion 2.5.1 Pointer Gestures. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a></p> 
+                  <p>この達成基準で扱われるドラッグ動作は、始点と終点だけが重要となるポインタの操作である。ポインタがターゲットに接触すると、ポインタがターゲットから外れるまで、ドラッグ動作の方向はインタラクションに考慮されない。ドラッグ動作には中間点がないため、ドラッグ動作は、どの方向にも進むことができる。軌跡ベースのジェスチャは、達成基準 2.5.1 ポインタ ジェスチャで扱われる。詳細については、<a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">解説書 達成基準 2.5.1 ポインタ ジェスチャ</a>を参照。</p> 
                   
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Users who struggle with performing dragging movements can still operate an interface
-                     with a pointer interface.
+                  <li>ドラッグ動作に苦労する利用者でも、ポインタインタフェースを使用してインタフェースを操作できる。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>A map allows users to drag the view of the map around, and the map has up/down/left/right
-                     buttons to move the view as well.
+                  <li>マップでは、利用者がそのマップのビューをあらゆる方向にドラッグして移動でき、そのマップにはビューを移動するための上下左右のボタンもある。
                   </li>
                   
-                  <li>A sortable list of elements may, after tapping or clicking on a list element, provide
-                     adjacent controls for moving the element up or down in the list by simply tapping
-                     or clicking on those controls.
+                  <li>要素の並べ替え可能なリストは、リスト要素をタップ又はクリックした後、それらのコントロールをタップ又はクリックするだけで要素をリスト内で上下に移動するための隣接するコントロールを提供してもよい。
```

Comment:
「それらのコントロールを」の「それら」が、日本語の語順の場合わかりにくいかもです。意訳気味になりますが、以下はいかがでしょうか？

```suggestion
                  <li>要素の並べ替え可能なリストは、リスト要素をタップ又はクリックした後、隣接するコントロールを提供して、それらのコントロールをタップ又はクリックするだけで要素をリスト内で上下に移動できるようにしてもよい。
```

Reply:
@caztcha 

とすると、あるいはこうでしょうか…？

```suggestion
                  <li>要素の並べ替え可能なリストは、リスト要素をタップ又はクリックした後に、隣接するコントロールを提供し、それらのコントロールをタップ又はクリックするだけで要素をリスト内で上下に移動できる。
```

Reply:
ありがとうございます！ @momdo さん案のほうが、原文に沿っていると思います。